doc/releases/nautilus.rst

```diff
@@ -84,6 +84,28 @@ Major Changes from Mimic
 
 - *CephFS*:
 
+  * MDS stability has been greatly improved for large caches and
+    long-running clients with a lot of RAM. Cache trimming and client
+    capability recall is now throttled to prevent overloading the MDS.
+  * CephFS may now be exported via NFS-Ganesha clusters in environments managed
+    by Rook. Ceph manages the clusters and ensures high-availability and
+    scalability.
+  * MDS standby-replay configuration is now done through a file system setting.
+    Config options for standby configuration are now obsolete.
+  * MDS now supports dropping its cache which concurrently asks clients
+    to trim their caches.
+  * It is now possible to check the progress of an on-going scrub in the MDS.
+    Additionally, a scrub may be paused or aborted.
+  * A new interface for creating volumes is provided via the ``ceph volume``
+    command-line-interface.
+  * A new cephfs-shell tool is available for manipulating a CephFS file
+    system without mounting.
+  * CephFS-related output from ``ceph status`` has been reformatted for brevity,
+    clarity, and usefulness.
+  * Lazy IO has been revamped. It be turned on by client using the new CEPH_O_LAZY flag to
+    the ``ceph_open`` API or via the config option ``client_force_lazyio``.
+  * CephFS file system can now be brought down rapidly via the ``ceph fs fail``
+    command.
```

Comment:
Side note: it would be *really* nice to have something definitive written up about what lazy IO actually _is_ and how to use it. What differences should I expect when this is enabled (particularly on cache coherency), and how do I mitigate the downsides of those effects? Some testcases too, while I'm wishing. ;)

Reply:
TODO: http://tracker.ceph.com/issues/38729